union.py

```diff
@@ -20,7 +20,7 @@ def union(self):
             write_file.writefile(self.result, result_data[0])
             return [self.list_1[1], self.list_2[1], result_data[1]]
         else:
-            raise Exception("Make sure the Give inpout files has the data ...")
+            raise Exception("Make sure, the Given input files has the data ...")
```

Comment:
grammatical issues

Reply:
**Fixed to** 
- `raise Exception("Make sure, the given input files has the data ...")`